test/models/category_test.rb

```diff
@@ -0,0 +1,54 @@
+require "test_helper"
+
+describe Category do
+  let(:food_category) { categories(:food) }
+
+  describe "relations" do
+
+    it "has products" do
+      food_category.must_respond_to :products
+    end
+
+    it "can have many products" do
+      food_category.products << products(:chocolate)
+      food_category.products.first.must_be_kind_of Product
+
+      food_category.products << products(:tears)
+      food_category.products.count.must_equal 2
```

Comment:
you should have a `food_category.save` here ensuring that no validations etc get broken.